### ISSUE TITLE
validate html in tests

### ISSUE THREAD

**Issue by mccalluc** [WGBH/stock-sales-2#67]

**Comment by mccalluc** [comment]
@afred: this exists: https://github.com/ericbeland/html_acceptance
... not working for me off the bat, but it can probably work. It does depend on command-line tidy for the heavy lifting, though. Thoughts about this gem, versus the approach taken in AAPB? (Add some optional closing tags, and then validate as XML.)

**Comment by afred** [comment]
@mccalluc yeah maybe. It looks like you can pass in arbitrary options to the underlying `tidy` command through the constructor, e.g. `HTMLAcceptance.new('/tmp/validation', config_file: "tidy.conf")`. If that's the case, then you should be able to specify a config file that defines the strictness. There are a lot of options if you type `tidy -help-config`.
